partner_communication_switzerland/models/contracts.py

```diff
@@ -266,8 +266,33 @@ def _send_reminders_for_birthday_in_1day_or_2months(self):
         tomorrow = today + relativedelta(days=1)
         sponsorships_with_birthday_tomorrow = \
             self._get_sponsorships_with_child_birthday_on(tomorrow)
+
+        sponsorships_to_send_reminder = []
+
+        for sponsorship in sponsorships_with_birthday_tomorrow:
+            sponsorship_correspondences = self.env['correspondence'].search([
+                ('sponsorship_id', '=', sponsorship.id),
+                ('direction', '=', "Supporter To Beneficiary"),
+                ('sent_date', '!=', False),
+                ('sent_date', '<=', str(datetime.now())),
+                ('sent_date', '>=', str(datetime.now() - relativedelta(months=-2)))
```

Comment:
Replace `sent_date` with `scanned_date`. Only the last one is useful. != False and is less than today are useless.
str(date) is not correct, should be replaced by `fields.Date.to_string()`